src/components/Personalization/extractDecisions.js

```diff
@@ -56,8 +54,6 @@ const splitDecisions = (decisions, predicate) => {
   return [matchedDecisions, nonMatchedDecisions, decisions];
```

Comment:
No need to return decisions here as it is the same as the parameter that you passed in.

Reply:
yeah, good catch, thank you